e2e/connection_test.go

```diff
@@ -39,14 +40,14 @@ func TestClientConnectsToNode(t *testing.T) {
 	err = tequilApi.Unlock(identity.Address, "")
 	assert.NoError(t, err)
 
-	registrationData, err := tequilApi.RegistrationStatus(identity.Address)
+	registrationData, err := tequilApi.IdentityRegistrationStatus(identity.Address)
```

Comment:
Asserting just fatal errors, I would assert structure of registrationData here

Reply:
I can assert only registration fact from this structure as its always new identity being generated.